### DISCUSSION TITLE
Fails to past clipboard, if config has p set to paste_clipboard_before

### DISCUSSION THREAD

**Discussion by punkbit** [helix-editor/helix#8819] [Troubleshooting]
### Summary

When "p" to "paste clipboard" fails if the config has "p" set to "paste_clipboard_before". 

### Reproduction Steps

 

Here's the asciinema:
https://asciinema.org/a/xm9mVAAvNKrL2Qt9vcb5YoUTE

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

macOS

### Terminal Emulator

kitty 0.27.0

### Installation Method

brew

### Helix Version

helix 23.10 (f6021dd0)

**Comment by gabydd** [comment]
``paste_clipboard_before`` uses your system clipboard you probably want ``paste_before`` that's what ``P`` is mapped to
